shoop/testing/management/commands/shoop_populate_mock.py

```diff
@@ -26,7 +26,8 @@ def handle(self, *args, **options):
         translation.activate(settings.LANGUAGES[0][0])
         superuser_name = options.get("with_superuser")
         if superuser_name:
-            user = get_user_model().objects.filter(username=superuser_name).first()
+            filter_params = {get_user_model().USERNAME_FIELD: superuser_name}
+            user = get_user_model().objects.filter(**filter_params).first()
             if not user:
                 user = get_user_model().objects.create_superuser(
                     username=superuser_name,
```

Comment:
Does `create_superuser` still work with the hardcoded `username` parameter or should it be changed too?